script.sql.txt

```diff
@@ -0,0 +1,173 @@
+CREATE DATABASE  IF NOT EXISTS `highschool` /*!40100 DEFAULT CHARACTER SET utf8 COLLATE utf8_bin */;
+USE highschool;
+CREATE TABLE STUDENT (
+	id_student INTEGER NOT NULL,
+	first_name VARCHAR(30) NOT NULL,
+	last_name VARCHAR (30) NOT NULL,
+	date_of_birth  DATE NOT NULL,
+	CONSTRAINT PK_STUDENT PRIMARY KEY (id_student)
+);
+
+CREATE TABLE TEACHER (
+	id_teacher INTEGER NOT NULL,
+	first_name VARCHAR(30) NOT NULL,
+	last_name VARCHAR(30) NOT NULL,
+	CONSTRAINT PK_TEACHER PRIMARY KEY (id_teacher)
+);
+
+CREATE TABLE COURSE (
+	id_course INTEGER NOT NULL,
+    course_name VARCHAR(60)  NOT NULL,
+	assigned_teacher INTEGER NOT NULL,
+	hours_by_week INTEGER NOT NULL,
+	time_start TIME NOT NULL,
+    time_end TIME NOT NULL,
+    day_of_week INTEGER NOT NULL,
+	CONSTRAINT PK_COURSE PRIMARY KEY (id_course)
+); 
+
+ALTER TABLE COURSE ADD CONSTRAINT FK_COURSE_TEACHER FOREIGN KEY (assigned_teacher)
+REFERENCES TEACHER (id_teacher) ON DELETE CASCADE ON UPDATE CASCADE;
+
+CREATE TABLE DAY (
+	id_day INTEGER NOT NULL,
+    day_name VARCHAR(60)  NOT NULL,
+	CONSTRAINT PK_DAY PRIMARY KEY (id_day)
+);
+
+ALTER TABLE COURSE ADD CONSTRAINT FK_COURSE_DAY FOREIGN KEY (day_of_week)
+REFERENCES DAY (id_day) ON DELETE CASCADE ON UPDATE CASCADE;
+
+
+CREATE TABLE STUDENT_REALIZED_COURSE (
+  note_1 INTEGER NOT NULL,
+  note_2 INTEGER NOT NULL,
+  note_3 INTEGER NOT NULL,
+  final_note REAL NOT NULL,
+  fk_student INTEGER NOT NULL,
```

Comment:
The prefix _**fk**_ for columns name is confuse. FK is used for name of FOREING KEYs.